index.js

```diff
@@ -0,0 +1,42 @@
+//select the tv button element.
+//create eventListener to initiate function when tv button is clicked.
+document.querySelector("#new-list").addEventListener("click", createList)
+
+//function to create a new list onclick.
+function createList() {
+
+    //create outer div to contain list.
```

Comment:
very neat code 😄 

Thank you for adding this JavaScript code for creating a new list ✨